data_summaries/MetaMathQA.json

```diff
@@ -0,0 +1,52 @@
+{
+    "meta_math_qa": {
+        "Unique Dataset Identifier": "meta_math_qa",
+        "Dataset Name": "meta_math_qa",
+        "Paper Title": "MetaMath: Bootstrap Your Own Mathematical Questions for Large Language Models",
+        "Dataset URL": "https://huggingface.co/datasets/meta-math/MetaMathQA",
+        "GitHub URL": "https://github.com/meta-math/MetaMath",
+        "Hugging Face URL": "https://huggingface.co/datasets/meta-math/MetaMathQA",
+        "Papers with Code URL": "",
```

Comment:
https://paperswithcode.com/paper/metamath-bootstrap-your-own-mathematical

Reply:
Tried adding this previously, but the tests failed since the highlighted part should be dataset and not paper according to the test. paperswithcode.com/`paper`/metamath-bootstrap-your-own-mathematical

Reply:
Good catch -- you're right.